src/metabase/api/search.clj

```diff
@@ -67,7 +66,7 @@
    table_db_id                         [:maybe ms/PositiveInt]
    models                              [:maybe (ms/QueryVectorOf search/SearchableModel)]
    filter_items_in_personal_collection [:maybe [:enum "only" "exclude"]]
-   context                             [:maybe [:enum "search-bar" "search-app" "command-palette"]]
+   context                             [:maybe [:enum "search-bar" "search-app" "command-palette" "entity-picker"]]
```

Comment:
I don't see `context` used anywhere else in this function aside from the removed code. Can it be removed as an arg?

Reply:
It probably could be. It's still doing some validation work to ensure the value is correct, but I suppose that could probably happen on the FE as well.